app/Http/Controllers/Admin/BusinessController.php

```diff
@@ -136,8 +136,7 @@ private function renderEdit(Business $business, $errors)
         $publishingStatuses = PublishingStatus::values();
         $authorizedStatuses = $publishingStatuses;
 
-
-        if (auth()->user()->getRole() === 4) {
+        if (auth()->user()->can('update', $business)) {
```

Comment:
I don't think this will work - it is going to set all users to only have PublishingStatus::DRAFT and READY_FOR_REVIEW.

SuperAdmins and RegionalAdmins should retain the other publishing statuses.

Reply:
Agree.  I've introduced a separate 'publish' permission.  See what you think.

This isn't tested yet - want to make sure I am doing the right code first.